services/google_service.js

```diff
@@ -1,32 +1,14 @@
-// const fetch = require('node-fetch');
-//
-// class GoogleService {
-//
-//   constructor(location) {
-//     this.location = location;
-//     this.lattitudeAndLong = null;
-//   }
-//
-//   getLatAndLng() {
-//     console.log(this.location);
-//     let location  = this.location;
-//     let googleApiKey = process.env.GOOGLE_API_KEY;
-//
-//     let url =`https://maps.googleapis.com/maps/api/geocode/json?address=${location}&key=${googleApiKey}`;
-//     fetch(url, { method: 'GET'})
-//       .then((response) => {
-//          return response.json();
-//     })
-//     .then((json) => {
-//       let googleGeocodeResponse = json;
-//       let latAndLng = Object.values(googleGeocodeResponse.results[0].geometry.location);
-//       console.log(`the lat and long of the searched for location are ${latAndLng}`);
-//       return latAndLng;
-//     })
-//     .catch((error) => {
-//        response.status(500).json({ error });
-//      });
-//   }
-//
-// }
-// module.exports = GoogleService;
\ No newline at end of file
+const fetch = require('node-fetch');
+
+async function getLatAndLongFromGoogle(location) {
+  const googleApiKey = process.env.GOOGLE_API_KEY;
+  const googleApiUrl =`https://maps.googleapis.com/maps/api/geocode/json?address=${location}&key=${googleApiKey}`;
+
+  let response = await fetch(googleApiUrl);
+  let googleGeocodeResponse = await response.json();
```

Comment:
this await may not be necessary?

Reply:
Thanks for that! Kayla from the front end told me to try using the await here too but i'll try it without that too! These promises are confusing!

Reply:
Update: I tried it without the await and it doesn't render the weather json correctly. It turns out that actually needs to be have an await - I think because it is being used in a map but I'm not 100% on that.

Reply:
@BabsLabs  Oh okay! Thanks for letting me know!